ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py

```diff
@@ -1,300 +1,300 @@
-    if is_null_or_empty(chap_user_name):
-        return None
-    else:
-        resp = client_obj.chap_users.get(name=chap_user_name)
-        if resp is None:
-            raise Exception(f"Invalid value for chap user {chap_user_name}")
-        return resp.attrs.get("id")
-
-
-def get_pe_id(client_obj, pe_name):
-    if is_null_or_empty(pe_name):
-        return None
-    else:
-        resp = client_obj.protocol_endpoints.get(name=pe_name)
-        if resp is None:
-            raise Exception(f"Invalid value for protection endpoint {pe_name}")
-        return resp.attrs.get("id")
-
-
-def get_snapshot_id(client_obj, vol_name, snap_name):
-    if is_null_or_empty(vol_name) or is_null_or_empty(snap_name):
-        return None
-    else:
-        resp = client_obj.snapshots.get(vol_name=vol_name, name=snap_name)
-        if resp is None:
-            raise Exception(f"No snapshot with name '{snap_name}' found for volume {vol_name}.")
-        return resp.attrs.get("id")
-
-
-def get_replication_partner_id(client_obj, replication_partner_name):
-    if is_null_or_empty(replication_partner_name):
-        return None
-    else:
-        resp = client_obj.replication_partners.get(name=replication_partner_name)
-        if resp is None:
-            raise Exception(f"Invalid value for replication partner {replication_partner_name}")
-        return resp.attrs.get("id")
-
-
-def get_volcoll_or_prottmpl_id(client_obj, volcoll_name, prot_template_name):
-    if is_null_or_empty(volcoll_name) and is_null_or_empty(prot_template_name):
-        return None
-    if is_null_or_empty(volcoll_name) is False and is_null_or_empty(prot_template_name) is False:
-        raise Exception("Volcoll and prot_template are mutually exlusive. Please provide either one of them.")
-    else:
-        if volcoll_name is not None:
-            resp = get_volcoll_id(client_obj, volcoll_name)
-            if resp is None:
-                raise Exception(f"Invalid value for volcoll: {volcoll_name}")
-        elif prot_template_name is not None:
-            resp = get_prottmpl_id(client_obj, prot_template_name)
-            if resp is None:
-                raise Exception(f"Invalid value for protection template {prot_template_name}")
-        return resp
-
-
-def get_downstream_partner_id(client_obj, downstream_partner):
-    if is_null_or_empty(downstream_partner):
-        return None
-    else:
-        resp = client_obj.replication_partners.get(name=downstream_partner)
-        if resp is None:
-            raise Exception(f"Invalid value for downstream partner {downstream_partner}")
-        return resp.attrs.get("id")
-
-
-def get_initiator_group_id(client_obj, ig_name):
-    if is_null_or_empty(ig_name):
-        return None
-    else:
-        resp = client_obj.initiator_groups.get(name=ig_name)
-        if resp is None:
-            raise Exception(f"Invalid value for initiator group {ig_name}")
-        return resp.attrs.get("id")
+#!/usr/bin/python
```

Comment:
Please change this to:
```
#!/usr/bin/env python
```
as per https://docs.ansible.com/ansible/2.9/dev_guide/testing/sanity/shebang.html

Reply:
thanks.. it worked.